engines/cge2/talk.cpp

```diff
@@ -254,6 +261,14 @@ InfoLine::InfoLine(CGE2Engine *vm, uint16 w, ColorBank color)
 }
 
 void InfoLine::update(const char *text) {
+	Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
+    if (text) {
+        if (lastText != text) {
```

Comment:
this looks suspicious. `lastText != text` is no string compare. It might be that you are comparing a buffer pointer with different content since the last call - but as the same buffer was used, the check didn't evaluate to true. It would imo be better to perform a string compare here.